components/collective-page/graphql/queries.js

```diff
@@ -53,6 +53,15 @@ export const getCollectivePageQuery = gql`
           all
         }
       }
+      memberOf(role: "CONNECTED_COLLECTIVE") {
```

Comment:
To make that clearer, I'd suggest aliasing the property (the matching prop will be renamed in `data`):

```suggestion
      connectedTo: memberOf(role: "CONNECTED_COLLECTIVE") {
```